kerrokantasi/tests/test_admin.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+from django.contrib.admin import AdminSite
+from django.contrib.auth import get_user_model
+
+from kerrokantasi.admin import UserAdmin
+
+
+@pytest.fixture
+def user_admin():
+    return UserAdmin(model=get_user_model(), admin_site=AdminSite())
+
+
+@pytest.mark.django_db
+def test_user_admin_admin_in_organizations_field_no_orgs(user_admin, user_factory):
+    user = user_factory()
```

Comment:
Just FYI, since you've got pytest factoryboy in use, you should also be able to just call for the `user` fixture instead of creating a user with the `user_factory`